backend/internal/adapter/persistence/queries/user.sql

```
@@ -125,11 +114,12 @@ UPDATE users SET is_active = $2, updated_at = now() WHERE id = $1;
 -- 氏名だけを更新する。0 件なら対象の user が存在しない（呼び出し側が not-found にする）。
 UPDATE users SET name = $2, updated_at = now() WHERE id = $1;
 
--- name: UpdateUserRoleID :execrows
+-- name: UpdateUserRole :execrows
 -- 役割だけを更新する（誰がどの役割になれるかの判定は usecase 側の仕事）。
 -- 0 件なら対象の user が存在しない（呼び出し側が not-found にする）。昇格が 1 行も
 -- 当たっていないのに成功を返すと、権限が上がったつもりの利用者が生まれる。
-UPDATE users SET role_id = $2, updated_at = now() WHERE id = $1;
+-- 未知のロール名は ck_users_role が弾く（黙って別ロールへ倒さない）。
+UPDATE users SET role = $2, updated_at = now() WHERE id = $1;
```

SQLFluff (4.3.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)